docker-compose.yml

```diff
@@ -12,15 +12,26 @@ services:
       - "50051:50051"
```

Comment:
You need to add http_port: 8082 here

Reply:
I can, but not a must, since both services share same `opi` network they can reach each other without exporting port, but I will add if somebody wants to send requests from host network as well, thanks